deployment/route96/templates/deployment.yaml

```
@@ -0,0 +1,91 @@
+    metadata:
+      labels:
+        app.kubernetes.io/name: blossom
+        app.kubernetes.io/part-of: blossom
+        app.kubernetes.io/managed-by: Helm
+    spec:
+      containers:
+      - name: my-container
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy | default "Always" }}
+        env:
+          - name: APP__LISTEN
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)